### DISCUSSION TITLE
Pas de catégorie "Scolaire secondaire" ou "Scolaire maternelle" sur la page de création d'une cantine

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1028] [Idées]
Actuellement, dans le menu déroulant "Secteur d'activité" de la page "Nouvelle cantine" sont proposées :

- "Scolaire" : pour les écoles maternelles, les collèges et les lycées
- "Scolaire primaire" : pour les écoles primaire

=> Je propose donc :
a. de supprimer la catégorie "Scolaire primaire" pour rassembler les écoles maternelles, les écoles primaires, les collèges et les lycées dans la seule catégorie "Scolaire"
OU
b. de créer 3 nouvelles catégories à la place de "Scolaire" : "Scolaire – Maternelle", "Scolaire – Collège" et "Scolaire – Lycée"

Par souci de simplicité pour l'utilisateur (je pense qu'il vaut mieux éviter les menus déroulants à rallonge) et d'homogénéité (on ne distingue par les ESMS par sous-secteur par exemple : "Hôpital", "EHPAD", "Établissement d'aide social", etc.), j'ai une préférence pour l'option a.

<img width="566" alt="Capture d’écran 2022-02-08 à 09 32 25" src="https://user-images.githubusercontent.com/93709421/152952222-89aee338-ca63-4561-a367-df067107b519.png">

**Comment by alemangui** [comment]
D'accord avec JB, je serais pour garder cette liste aussi simple que possible. @valeriemerle c'est quoi le besoin par rapport au split de Scolaire / Scolaire Primaire ? Car là il y a découpe entre les deux catégories

**Reply by valeriemerle** [reply]
je milite plutot pour l'option b. C'est moi qui est commencé à modifier ce champ mais pas complément...il manque effectivement "Scolaire – Collège" et "Scolaire – Lycée". Cette séparation est motivée par le fait que le secteur scolaire est très hétérogène du point de vue des types d'acteurs et de leurs fonctionnement. D'autre part dans la future exploitation des données la séparation entre ces trois sous secteur rendra plus aisée les analyses et la comparaison entre sous secteurs.

**Reply by valeriemerle** [reply]
Je pense aussi que d'autres sectorisations seraient nécessaires notamment dans le secteur médico social trés hétérogene comportant des etablissements d'accueil de jeunes et des maisons de retraite par ex. A réfléchir avec Pauline. Attention ce secteur ne comprends pas les hopitaux et cliniques, rattachés au secteur "médical".

**Reply by alemangui** [reply]
Merci Val. Ça serait plus adapté de faire le changement en démo ou staging si c'est un test ou une étape intermédiaire, sur prod tous les utilisateurs sont obligés de remplir les secteurs. 

D'ailleurs t'entends quoi par sous-secteur ? Il faudra bien équilibrer la simplicité du parcours usager des besoins des stats.

**Reply by alemangui** [reply]
À part la production de stats y-a-t'il une raison à fragmenter les secteurs comme ça ? Qu'est-ce qu'on fait avec ? Ou plutôt, quel problème rencontrons-nous du fait que le secteur médico-social est groupé dans un secteur chez nous ?

**Comment by valeriemerle** [comment]
Il serait intéressant, pour éviter des confusions pour les utilisateurs lors de leur "inscription" de clarifier ou e mieux définir les secteurs par des info bulles par exemple sur les secteurs. On constate que des etablissements se sont inscris dans le secteur entreprise alors que ce sont des administrations et vis et versa. Par ailleurs est ce possible de distinguer un secteur principal et un secteur secondaire? plus facile ainsi de trier les etablissements dans l'analyse des données? par exemple une cuisine centrale d'une commune fabrique pour 85% pour des cantines scolaire et peut aussi sur les 15ù restant fournir des repas à des personnes agées en portage à domicile ou pour une maison de retraite ou pour des creches.

**Reply by alemangui** [reply]
Pour cet exemple en particulier, la cuisine centrale à terme n'aurait pas de compte chez Ma Cantine (vu que ce n'est pas eux qui télédéclarent ni publient). Mais bon, ça c'est une autre discussion ici : https://github.com/betagouv/ma-cantine/discussions/1029

Ce n'est pas encore clair pour moi quel problème on a qui serait résolu par une division et hiérarchisation des secteurs. Qu'est-ce que les utilisateurs gagnent de pouvoir spécifier _x_ niveaux des secteurs avec une meilleure précision ? Comment ça nous aide dans nos objectifs ? J'ai crains que ça soit une sur-complexification pour produire des stats à terme, et que le prix à payer soit un parcours utilisateur plus compliqué, des doutes sur quel secteur choisir (ce qui est déjà le cas avec le doublon scolaire), et un coût plus élévé de maintenance suite à cette granularité.
